### DISCUSSION TITLE
Create file from file link in document

### DISCUSSION THREAD

**Discussion by MyraBaba** [nvim-neorg/neorg#459] [General]
Hi,

I cant find a clear information how I can create file from file link ?  
ie : {:file:* This is a link location}. create file.norg 

and is the only way to add norg file to e file.norg or there is a command ?

Best

**Comment by max397574** [comment]
this should be possible ig
just create the file when link not found

**Comment by MyraBaba** [comment]
@max397574 

clicking the link creating the file. I found :)

**Reply by max397574** [reply]
oh
so this is already the case?
I just wanted to suggest an implementation
